src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/api/OppfolgingstilfelleSystemApiSpek.kt

```diff
@@ -147,17 +128,73 @@ class OppfolgingstilfelleSystemApiSpek : Spek({
                             val oppfolgingstilfellePersonDTO: OppfolgingstilfellePersonDTO =
                                 objectMapper.readValue(response.content!!)
 
-                            oppfolgingstilfellePersonDTO.personIdent shouldBeEqualTo oppfolgingstilfelleBit.personIdentNumber.value
+                            oppfolgingstilfellePersonDTO.personIdent shouldBeEqualTo kafkaSyketilfellebitRelevantVirksomhet.fnr
 
                             val oppfolgingstilfelleDTO =
                                 oppfolgingstilfellePersonDTO.oppfolgingstilfelleList.first()
 
                             oppfolgingstilfelleDTO.virksomhetsnummerList.size shouldBeEqualTo 1
-                            oppfolgingstilfelleDTO.virksomhetsnummerList.first() shouldBeEqualTo oppfolgingstilfelleBit.virksomhetsnummer
+                            oppfolgingstilfelleDTO.virksomhetsnummerList.first() shouldBeEqualTo kafkaSyketilfellebitRelevantVirksomhet.orgnummer
+
+                            oppfolgingstilfelleDTO.arbeidstakerAtTilfelleEnd shouldBeEqualTo true
+                            oppfolgingstilfelleDTO.start shouldBeEqualTo kafkaSyketilfellebitRelevantVirksomhet.fom
+                            oppfolgingstilfelleDTO.end shouldBeEqualTo kafkaSyketilfellebitRelevantVirksomhet.tom
+                        }
+                    }
+                    it("should not return future oppfolgingstilfelle when using get in api") {
```

Comment:
Nå har vi akkurat samme test to ganger? Da virker det litt som at noe kunne vært skilt ut og samlet i én fil fra disse api-filene våre?

Reply:
Ja, men det er jo to forskjellige api'er (det ene med veileder-token, det andre med system-token). Synes det er greit å ha en test for hver av dem selv om det treffer en del av den samme koden.